src/config/emailContent.js

```diff
@@ -0,0 +1,27 @@
+const CONTACT_MESSAGE_FIELDS = {
+  first_name: 'First Name: ',
+  last_name: 'Last Name: ',
+  mobile_phone: 'Phone Number: ',
+  email: 'Email: ',
+  dob: 'Date of Birth: ',
+  requested_date: 'Requested Date: ',
+  requested_time: 'Requested Time: ',
+  appointment_type: 'Appointmet Type: ',
+  description: 'Description: ',
+  is_emergency: 'Is this an emergency? ',
+};
+
+export const generateEmailContent = (data) => {
+  const stringData = Object.entries(data).reduce((str, [key, val]) => {
+    return (str += `${CONTACT_MESSAGE_FIELDS[key]}: \n${val} \n\n`);
+  }, '');
+
+  const htmlData = Object.entries(data).reduce((str, [key, val]) => {
```

Comment:
It's really nice to see how you have implemented the whole functionality. I have a few suggestions. 
1. Right now, the email's html content appears like this on desktop:
<img width="578" alt="image" src="https://github.com/Full-Stack-Collective/connectient/assets/27571141/4c8e1a7d-9d19-4089-9830-528b69e17511">

That's great. But, it will be more easy to comprehend for the patient if the heading and values appear in the same line. The email content also will look more cohesive. It should be some easy css tweaks.

2. Also, we could make the body little welcoming. Including a simple 'Hi <Patient Name>' with a message like 'Thanks for using Connectient Appointment Request System. Your appointment has been requested successfully. Our staff will give you a call on the contact number provided within 48 hours to confirm the same. Please, keep an eye.' Something like that. 

3. We should also provide the contact info of the clinic in case the user wants to change something or for any other problems.

Reply:
Thanks for all of these suggestions! I had just done this as a test to see how this would work. I can definitely work on making the email itself prettier and more informative.